test/static.test.js

```diff
@@ -374,6 +375,353 @@ t.test('register /static', (t) => {
   })
 })
 
+t.test('register /static', (t) => {
```

Comment:
This was to make sure we tested at least once for the 'HEAD' method, not strictly related to the PR